packages/shared/src/components/ScrollToTopButton.tsx

```diff
@@ -16,6 +16,8 @@ const baseStyle: CSSProperties = {
 export default function ScrollToTopButton(): ReactElement {
   const [show, setShow] = useState(false);
   const isLaptop = useViewSize(ViewSize.Laptop);
+  const isTablet = useViewSize(ViewSize.Tablet);
+  const toucheDeviceSize = isTablet ? ButtonSize.Large : ButtonSize.Small;
```

Comment:
Pardon my english, what does touche mean in this case? 😅 
Personally I would do all conditions in one also for the laptop, and use just the variable below.

Reply:
Ah, should be `touch`. Lol, I misspelled touch device.

> Personally I would do all conditions in one also for the laptop, and use just the variable below.

Do you mean in one line? I think it can become confusing, and our lint doesn't accept multiple ternaries in a single line. Or do you mean something else?

Reply:
Yeah, that's what I ment, but ok like this as well. Just feels weird we have a variable for mobile tablet size but check condition separately for laptop

Reply:
Yeah, same sentiments. I even thought, we can maybe use a function.

```
function getSize() {
  if (laptop) return xl;
  if (tablet) return l;

  return s;
}
```

Wdyt?

Reply:
Yeah looks good